### DISCUSSION TITLE
糖之妙效 | /cn/2022/07/sugar-effect/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1537] [Comments]
酸酸的桑葚
浸入醇甜的酒酿
[&hellip;] 清苦的苦瓜
敷上蜂蜜
[&hellip;] 腥的肉丝　抖进糖砂
大火中浓郁的香
[&hellip;] 无味的硬花生　搅进糖稀
停不下来的嘎嘣
[&hellip;] 煮开花的绿豆　冻进糖水
吮吸千百遍的暑假冰棒
[&hellip;] 咸的酱油酸的醋汁　两勺糖进去
三百颗荔枝出来
[&hellip;] 啊？你说啥？辛辣？
无妨　那也无妨
考不倒我
本厨 &hellip;

https://yihui.org/cn/2022/07/sugar-effect/

**Comment by ZaiyanHere** [comment]
心中有糖的这一刻，我的眼睛里闪的其实是泪光，是开心和感激的泪光。
